e2e/process-services-cloud/task-header-cloud.e2e.ts

```diff
@@ -143,4 +143,10 @@ describe('Task Header cloud component', () => {
             .toEqual(subTask.entry.parentTaskId === null ? '' : subTask.entry.parentTaskId);
     });
 
+    it('[C307032] Should display the appropriate title for the unclaim option of a Task', async () => {
```

Comment:
For my point of view, we should check the title of a consequence of a use case. 
For example:
As a user, I should be able to release a task claimed by me.
This means that the button release should be rendered into the page

Reply:
I agree, but as there is currently an outstanding bug here, ADF-4315, expanding this test to check the end to end flow and assert that is Task is displayed with filters  Status = Created and Assignee is empty, will be done as part of ADF-4315.